apps/aecore/src/aec_conductor.erl

```diff
@@ -227,6 +227,7 @@ handle_call({post_block, Block},_From, State) ->
     {reply, Reply, State1};
 handle_call(stop_mining,_From, State = #state{ consensus = Cons }) ->
     epoch_mining:info("Mining stopped"),
+    aec_block_generator:stop_generation(),
```

Comment:
Please ensure that merging to master this does not introduce [double stopping](https://github.com/aeternity/epoch/blob/28be867f7f3e7bfb66eaa3c0a487933815d83f5a/apps/aecore/src/aec_conductor.erl#L230-L231).